subsys/net/openthread/rpc/common/ot_rpc_ids.h

```diff
@@ -14,7 +14,7 @@ enum ot_rpc_cmd_client {
 
 /** @brief Command IDs accepted by the OpenThread over RPC server.
  */
-enum ot_rpc_cmd_server {
+typedef enum ot_rpc_cmd_server {
```

Comment:
Why should it be type?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/16326/files#diff-8db7e3b76c5d84d7087e0da2235cd6bc59a0e6b6aac837367d7d1f7f2127d902R26  IHMO it's better to define a type def than utilize a `int` to store the value.

Reply:
As I already mentioned in @Damian-Nordic's PR just usage `enum ot_rpc_cmd_server` is sufficient and this is according to Zephyr's coding paradigm (https://github.com/nrfconnect/sdk-nrf/pull/16406#discussion_r1673925000). Creation of new type is not required.

Reply:
Agreed I removed type def from enum.